src/vs/workbench/common/editor/editorGroup.ts

```diff
@@ -332,7 +332,26 @@ export class EditorGroup extends Disposable {
 
 			// More than one editor
 			if (this.mru.length > 1) {
-				this.setActive(this.mru[1]); // active editor is always first in MRU, so pick second editor after as new active
+				const tabClosingOrder = this.configurationService.getValue('workbench.editor.tabClosingOrder');
```

Comment:
@SimonEggert I would prefer if this setting was put as variable similar to `private editorOpenPositioning` so that we do not have to compute it each time

Reply:
Thanks for the hint, I already thought there would be a better way to do this.